pkg/asset/manifests/infrastructure.go

```diff
@@ -144,6 +143,12 @@ func (i *Infrastructure) Generate(dependencies asset.Parents) error {
 		}
 	case vsphere.Name:
 		config.Status.PlatformStatus.Type = configv1.VSpherePlatformType
+		// Needs https://github.com/openshift/api/pull/541
+		// config.Status.PlatformStatus.VSphere = &configv1.VSpherePlatformStatus{
```

Comment:
@mtnbikenc 
Can you add a check here if the strings are empty.  If so we will want the PlatformStatus.VSphere to be nil.
MCO then won't render the bootstrap manifests that is required only for IPI.

Reply:
See https://github.com/openshift/machine-config-operator/pull/1373/files#diff-d6f38878757202a810cfa8038e7ae122R272
for additional details.

Reply:
Updated.